### ISSUE TITLE
Unhandled exception: Tried to pull CO.AG Soundpack, crashed.

### ISSUE THREAD

**Issue by Pseudo-ku** [remyroy/CDDA-Game-Launcher#270]
* Description:

Trying to install the CO.AG Soundpack from the Repository, under the Soundpacks tab, leads to a crash.

As a sidebar, the URL supplied in the Details of the soundpack leads to the download page for the Hydroponics mod... but that may be a separate issue.

First time contributing an issue in a while, let me know if you need anything more. I'll respond as quick as I can until the issue is solved.
* Version: 1.3.18
* OS: Windows-7-6.1.7601-SP1 (64-bit)
* Type: `<class 'cddagl.win32.PathNotFoundException'>`
* Value: 
* Traceback:
```
  File "cddagl\ui.py", line 4477, in install_new
  File "cddagl\ui.py", line 4039, in __init__
  File "cddagl\win32.py", line 441, in get_downloads_directory
  File "cddagl\win32.py", line 434, in get_path

```

**Comment by remyroy** [comment]
Did you download the file *CO.AG SoundPack (v1.1) (02-Mar-19).zip* from the google drive? Did you select that file in the browser download window?

**Comment by Pseudo-ku** [comment]
I have not been able to download the .zip at all through the launcher. I ended up downloading it externally off of the Home Page supplied in the Details (remembering that the URL is incorrect). I had no idea there was a google drive repository for all the documents.

**Comment by remyroy** [comment]
If you click on the install button for the CO.AG Soundpack, don't you see this window?

![browser-download](https://user-images.githubusercontent.com/303593/56859231-e2d11280-6955-11e9-8307-c8b530cca07e.png)

**Comment by Pseudo-ku** [comment]
After clicking on the Install button the window does not show at all, it just goes straight to crashlog with the info posted in the issue report.

**Comment by remyroy** [comment]
The launcher tries to find your default download directory using SHGetKnownFolderPath but it seems to fail on your system. let me try to catch this exception and perform a workaround.

**Comment by remyroy** [comment]
This should be solved with e6e21c6bae6566d424a03e0e58c8edd315663710 .
